src/gamestate/guild.rs

```diff
@@ -164,9 +164,18 @@ impl Guild {
 
         for (offset, member) in self.members.iter_mut().enumerate() {
             member.battles_joined =
-                data.cfpget(64 + offset, "member joined", |x| x / 1000)?;
+                match data.csiget(445 + offset, "guild member battles joined", 0)? % 100 {
+                    0 => None,
+                    1 => Some(BattlesJoined::Defense),
+                    10 => Some(BattlesJoined::Attack),
+                    11 => Some(BattlesJoined::Both),
+                    x => {
+                        warn!("Unknown battles joined: {x}");
+                        None
+                    }
+                };
```

Comment:
Can this just be:
```rust
data.cfpget(445 + offset, "guild member battles joined", |x| x % 100)?;
```

With the enum changed to:

```rust
pub enum BattlesJoined {
    Defense = 1,
    Attack = 10,
    Both = 11,
}
```
Or is there a reason for the syntax change?